src/vm/methodtable.cpp

```diff
@@ -2300,6 +2300,19 @@ const char* GetSystemVClassificationTypeName(SystemVClassificationType t)
 };
 #endif // _DEBUG && LOGGING
 
+// Returns 'true' if the struct is passed in registers, 'false' otherwise.
+bool MethodTable::ClassifyEightBytes(SystemVStructRegisterPassingHelperPtr helperPtr, unsigned int nestingLevel, unsigned int startOffsetOfStruct, bool useNativeLayout)
+{
+    if (useNativeLayout)
+    {
+        return ClassifyEightBytesForNativeStruct(helperPtr, nestingLevel, startOffsetOfStruct, useNativeLayout);
```

Comment:
Nit: `ClassifyEightBytesWithNativeLayout` and `ClassifyEightBytesWithManagedLayout` might be clearer names for these methods.